src/org/openjump/core/rasterimage/AddRasterImageLayerWizard.java

```diff
@@ -263,175 +263,114 @@ private void addImage(WorkbenchContext context, Envelope envelope,
      * @throws IOException
      * @throws NoninvertibleTransformException
      */
-    protected Envelope getGeoReferencing(String fileName,
+     protected Envelope getGeoReferencing(String fileName,
             boolean alwaysLookForTFWExtension, Point imageDimensions,
-            WorkbenchContext context) throws IOException,
-            NoninvertibleTransformException {
-        double minx, maxx, miny, maxy;
-        Envelope env = null;
-
-        worldFileHandler = new WorldFileHandler(fileName,
-                alwaysLookForTFWExtension);
-
-        if (imageDimensions == null) {
-            // logger.printError("can not determine image dimensions");
-            context.getWorkbench()
-                    .getFrame()
-                    .warnUser(
-                            I18N.getInstance().get("org.openjump.core.rasterimage.AddRasterImageLayerWizard.can-not-determine-image-dimensions"));
-            return null;

```

Comment:
this is wrong. you added tabs to the indentation (spaces only before) and did some unneeded reformatting. i cannot see clearly what you changed here.

please fix the whole file. compare with previous version from branch main.

Reply:
I see the tabs+spaces. I will correct it

Should I close this pull request and open a new one?

Reply:
no. you can just push to the branch this PR is based on :)